[DOCFIX] Add doc on Presto and Iceberg

dbw9580 opened the pull request:
A new documentation article on how to query Iceberg tables with Presto and Alluxio.

HelloHorizon commented:
@dbw9580 we usually add prefix [DOCFIX] to skip unnecessary GitHub action tests for document update.

beinan commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,107 @@
+and configuration. Most of that guide apply for Iceberg workflows as well, and this document 
+covers the specific instructions for working with Iceberg tables.
+
+Before proceeding, make sure you have a working setup of Presto, Hive metastore and Alluxio, 
```
Looks like a bit overlapped with line 16

beinan commented on docs/en/compute/Presto-Iceberg.md:
````diff
@@ -0,0 +1,107 @@
+$ ${PRESTO_HOME}/bin/launcher restart
+```
+
+Also note that the same client jar file needs to be copied to Hive's `lib` directory.
````
Just wanna clarify, why do we need to put the alluxio jar to hive's lib? is it for hive metastore or hive?  Is this step mandatory?

dbw9580 replied:
What I really meant is that the jar needs to be on the classpath of Hive. I've always copied the jar to that directory, so I went with that. I changed it to align with the instructions from the [Hive article](https://docs.alluxio.io/ee/user/stable/en/compute/Hive.html#basic-setup).

> is it for hive metastore or hive?

For Hive Metastore.

> Is this step mandatory

From what I have exprienced without that jar, Presto will fail to create a new schema with location set to an Alluxio URI. The error is `java.io.IOException No FileSystem for scheme: alluxio`.

beinan replied:
I see, thank you for clarifying!

beinan commented on docs/en/compute/Presto-Iceberg.md:
````diff
@@ -0,0 +1,107 @@
+hive.metastore.uri=thrift://localhost:9083
+```
+
+Change the Hive connection URI to match your Hive setup.
````
To be more precise, I would say Hive-Metastore rather than Hive here.

beinan commented on docs/en/compute/Presto-Iceberg.md:
````diff
@@ -0,0 +1,107 @@
+Launch the Presto CLI client, and run the following statements:
+
+```sql
+CREATE SCHEMA iceberg_test;
````
nit: should we do 'use iceberg.default' first?  or `CREATE SCHEMA iceberg.iceberg_test;` ?

dbw9580 replied:
I accidentally left out the command to start the client with `--catalog iceberg`. This has been fixed in the last commit.

beinan replied:
Cool, using --catalog is great

beinan reviewed (commented):
Great write-up!  We just need such docs, thank you!  Just a couple of clarification questions.

beinan commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,107 @@
+and Presto can access data through Alluxio's filesystem interface. 
+The [guide]({{ '/en/compute/Presto.html' | relativize_url }}) mentions two approaches that 
+Presto can access data through Alluxio: via the filesystem interface, and via the catalog service. 
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
```
Might be out of scope of this doc, I'm just curious what kind of support needed for Iceberg in catalog service? Did we have any relevant discussions before？

dbw9580 replied:
I'm not particularly sure about this, but from what I know about the existing Hive support, we need to make SDS understand the Iceberg format so that SDS can mount the correct files, report statistics of the tables to Presto when it's generating query plans, etc.

beinan replied:
Gotcha, sounds good to me.

dbw9580 commented:
Addressed comments in the last commit, please take another look, thanks!

beinan reviewed (approved):
This doc looks good to me. @apc999 do you want to have a second look?

apc999 commented on docs/en/compute/Presto-Iceberg.md:
````diff
@@ -0,0 +1,115 @@
+
+Launch the Presto CLI client with the following command:
+
+```shell
````
```suggestion
```console
```

dbw9580 replied:
Done

apc999 commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,115 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
```
```suggestion
title: Running Presto on Iceberg Tables with Alluxio
```

apc999 commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,115 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
+nickname: Presto-Iceberg
```
```suggestion
nickname: Presto on Iceberg (Experimental)
```

apc999 commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,115 @@
+
+Presto has introduced support for [Iceberg tables](https://iceberg.apache.org/) in version 0.256.
+
+This document describes how to use Presto to query Iceberg tables through Alluxio.
```
Add a statement that this page is currently experimental?

apc999 commented on docs/en/compute/Presto-Iceberg.md:
```diff
@@ -0,0 +1,115 @@
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
+needs to access them through Alluxio's filesystem interface.
+
+## Install Alluxio client jar to Presto Iceberg connector
```
Let's have the outline like this (similar as Presto page):

<pre>

## Prerequisites
Put what you require

## Basic Setup
Describe the bare minimal setup to let users run this stack

## Examples
Put the query example

## Advanced Setup
Any advanced configuration (non default)
</pre>

apc999 reviewed (changes requested):
Let's have the outline like this (similar as Presto page):

<pre>

## Prerequisites
Put what you require

## Basic Setup
Describe the bare minimal setup to let users run this stack

## Examples
Put the query example

## Advanced Setup
Any advanced configuration (non default)
</pre>

dbw9580 commented:
@apc999 forgot to update the status of this PR: last commit addressed comments, also updated required Presto version to 0.257, since the fix for insertion failure is landed in that version recently. PTAL, thanks.

apc999 commented:
alluxio-bot, merge this please
